pkg/controller/account/account_controller.go

```diff
@@ -812,7 +812,7 @@ func CreateUserAccessKey(reqLogger logr.Logger, client awsclient.Client, userNam
 
 	result, err := client.CreateAccessKey(input)
 	if err != nil {
-		return &iam.CreateAccessKeyOutput{}, errors.New("Error creating access key")
+		return &iam.CreateAccessKeyOutput{}, fmt.Errorf("Error creating access key: %v", err)
 	}
 
 	if err != nil {
```

Comment:
What about this code? It is currently unreachable. Should this `controllerutils.LowAwsError(...)` go right before the `return` above?